src/main/java/org/ohdsi/circe/check/checkers/UnusedConceptsCheck.java

```diff
@@ -95,4 +101,14 @@ private boolean isConceptSetUsed(ConceptSet conceptSet, CriteriaGroup group) {
         return Objects.nonNull(criteriaList) ? Arrays.stream(criteriaList)
                 .map(c -> c.criteria).collect(Collectors.toList()) : Collections.emptyList();
     }
+
+    private List<Criteria> toCriteriaList(CriteriaGroup[] groups) {
+
+        List<Criteria> criterias = new ArrayList<>();
```

Comment:
Ah, yes here: 'criterias' isn't a word. The plural is 'criteria' the singular is 'criterion'.

Reply:
Thank you. I didn't know about it